py-polars/polars/internals/series/series.py

```diff
@@ -1977,20 +1978,21 @@ def top_k(self, k: int = 5, reverse: bool = False) -> Series:
         ----------
         k
             Number of elements to return.
-        reverse
+        descending
             Return the smallest elements.
 
         """
 
```

Comment:
```suggestion
return (
            pli.wrap_s(self._s)
            .to_frame()
            .select(pli.col(self._s.name()).top_k(k=k, descending=descending))
            .to_series()
        )

```

Reply:
This was it indeed, thank you. Had been going over the Rust code for hours, not realizing the Python decorator was the culprit.